### PR TITLE
Aw commercial properties on edition

### PR DESCRIPTION
**Pull request by AWare**
## What does this change?
Moves everything round in commercial properties
https://github.com/guardian/frontend/pull/21208

### CODE REVIEW

File: packages/frontend/lib/branding.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export interface Brand {
```
Comment (gtrufitt) [inline packages/frontend/lib/branding.ts]: Is this repeated on purpose?

File: packages/frontend/model/extract-capi.ts [hunk 2]
```diff
@@ -19,10 +19,10 @@ const apply = (input: string, ...fns: Array<(_: string) => string>): string => {
     return fns.reduce((acc, fn) => fn(acc), input);
 };
 
-const getEditionValue: (name: string) => Edition = name => {
+const getEditionValue: (name: string) => Edition | false = name => {
```
Comment (gtrufitt) [inline packages/frontend/model/extract-capi.ts]: Is false what we want over a default Edition?

File: packages/frontend/model/extract-capi.ts [hunk 3]
```diff
@@ -158,11 +140,8 @@ export const extract = (data: {}): CAPIType => {
     const editionLongForm = getString(data, 'config.page.edition', '');
 
     // Possible values for the editionId: "UK", "US", "AU", "INT"
-    const editionId = getEditionValue(
-        getString(data, 'config.page.editionId', ''),
-    );
-
-    if (editionId === undefined) throw new Error('edition id is undefined');
+    const editionId =
+        getEditionValue(getString(data, 'config.page.editionId', '')) || 'UK';
```
Comment (gtrufitt) [inline packages/frontend/model/extract-capi.ts]: Hmm, ok, so the default come from here. Also, should that default be `UK`, I feel like it's maybe `INT` in Frontend (But actually UK makes sense to me, just be good to know that's defo what we want)

Comment (AWare) [reply packages/frontend/model/extract-capi.ts]: I was respecting the previously set default here. But you're right that the choice should be deliberate.

File: packages/frontend/model/extract-capi.ts (hunk 2 above)
Comment (AWare) [reply packages/frontend/model/extract-capi.ts]: Yeah, just wanted the ability to distinguish when an edition is missing from a string.

File: packages/frontend/lib/branding.ts (hunk 1 above)
Comment (AWare) [reply packages/frontend/lib/branding.ts]: No.

**Pull request merged**